app/index.jsx

```diff
@@ -16,6 +16,14 @@ let App = React.createClass({
   changeSelectedElement: function (clickedLiIndex) {
     this.setState({ selectedElementIndex: clickedLiIndex })
   },
+  copyToClipboard: function () {
+    const textField = document.createElement('textarea')
```

Comment:
Don't do this in React.

I forgot to mention it, but you shouldn't be making any changes directly to the DOM. Changes to the DOM are done automatically every time a React component gets rendered. You should only modify state. I'll do my own PR to show how this can be done.